tests/modules/services/playerctld/basic.nix

```diff
@@ -0,0 +1,16 @@
+{ config, pkgs, ... }:
+
+{
+  config = {
+    services.playerctld.enable = true;
+
+    nmt.script = ''
+      serviceFile=home-files/.config/systemd/user/playerctld.service
+
+      assertFileExists "$serviceFile"
+      assertFileRegex "$serviceFile" 'BusName=org\.mpris\.MediaPlayer2\.playerctld'
+      assertFileRegex "$serviceFile" 'Type=dbus'
+      assertFileRegex "$serviceFile" 'ExecStart=.*/bin/playerctld'
```

Comment:
With the dummy package, it becomes possible to rewrite this as `assertFileContent ${pkgs.writeText "playerctld-test" '' file content ''}`

Reply:
That does look nicer